modules/users/manifests/sebastianschmieschek.pp

```diff
@@ -1,9 +0,0 @@
-# Creates the user schmie for Sebastian Schmieschek
-class users::sebastianschmieschek {
-  govuk_user { 'sebastianschmieschek':
-    fullname => 'Sebastian Schmieschek',
```

Comment:
You'll need to `ensure => absent` for the user to be removed from Puppet along with their home directory, otherwise their files hang around forever. https://docs.publishing.service.gov.uk/manual/removing-a-user-from-puppet.html

Reply:
🤦‍♂ I used to know that. Thanks Issy.

Changed in ed5779e